### ISSUE TITLE
3.1.2.4 Access to data

### ISSUE THREAD

**Issue by isedwards** [opencdms-project/project-technical-team#69]
This component involves a policy that governs how an organization allows access to data.

Some issues to consider are:
1. How is read-and-write access to climate data controlled?
2. What training is needed before staff are granted write access?
3. What types of controls are needed to monitor and approve changes to the climate record to facilitate data maintenance and error rectification?
4. How are any specific access constraints, such as security, contractual or commercial constraints, to be managed?

Note: The issue of open data is increasing in relevance for many national governments that have made open-data commitments. 

Therefore, organizations should have a clear understanding of, and a clear policy for, the provision of NMHS climate data in accordance with open data principles (see Wikipedia article on open data).

In summary, open data is about ensuring that data are freely available for reuse with no constraints apart from perhaps the requirement to acknowledge the data source and ensure that the data are made available under a share-alike agreement.
